src/components/TimerModal/Modal.module.scss

```diff
@@ -0,0 +1,77 @@
+.modalContainer {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100vw;
+    height: 100vh;
+    overflow: hidden;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: rgba(0, 0, 0, 0.3);
+    z-index: 100;
+}
+
+.playButtonDiv {
+    text-align: center;
+    margin: 12px 12px 12px 22px;
+    width: 20px;
+    height: 20px;
+    border-style: solid;
+    color: white;
+    border-width: 10px 0 10px 20px;
+    border-color: transparent transparent transparent white;
```

Comment:
what's this for? 
I don't think there should be a border for the play/pause button